src/thunderbots/software/grsim_communication/main.cpp

```diff
@@ -29,6 +31,8 @@ namespace
     std::vector<std::unique_ptr<Primitive>> primitives;
 
     Team friendly_team = Team(std::chrono::milliseconds(1000));
+
+    Ball ball;
```

Comment:
I'm not sure if this will compile? You'll probably want to provide some default arguments here, like:
`Ball ball = Ball(Point(), Vector());`